src/bindist/autodos7.bat

```diff
@@ -0,0 +1,6 @@
+@ECHO Off
+PATH D:\OPENDOS;D:\EMUBIN;D:\
+VERIFY OFF
+PROMPT [OpenDOS] $P$G
+SET OPENDOSCFG=D:\OPENDOS
+CALL D:\EMUBIN\UXMACROS.BAT
```

Comment:
I just looked over all the files in the emubin directory. It seems that uxmacros.bat is the only one that's useful. The rest are just old versions of the tools that are included with dosemu now. (Except for lancheck, but that seems outdated.)

Maybe uxmacros.bat could be just added to the dosemu repo and used with other dos versions as well that include a suitable doskey.